build/enc-dockerhub.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# - Run `$(command -v winpty) docker run --rm -it alpine sh -c "REPO='https://github.com/<user|organization>/<repo>'; USERNAME=<username>; PASSWORD=<password>; $(cat travis-enc.sh)"`
```

Comment:
Related to the comment about secure envvars above:

Reply:
A couple of additional notes on this topic:

- Encrypted environment variables in travis only work for own branches, not for PRs. This ensures that third parties won't be able to push potentially harmful images to the docker hub.
- In my fork I am using travis-ci.com, because they are migrating open-source projects that used to be hosted in travi-ci.org (see https://blog.travis-ci.com/2018-05-02-open-source-projects-on-travis-ci-com-with-github-apps). However, this repo still is active in travis-ci.org. I suggest to change this before creating the new hashes.

Reply:
> Therefore, you should be the one that does it, or you should temporally give write permissions in this repo to me.

You have **write** permissions to all repositories on this organisation 😄

> However, this repo still is active in travis-ci.org. I suggest to change this before creating the new hashes.

Just disabled it on the .org website. Although, on .com, it still says it is active on .org. Maybe we need to wait.

Reply:
> You have write permissions to all repositories on this organisation smile